normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -47,6 +47,10 @@ export class RecipeForm extends React.Component {
 
     return this.validateForm(combinedFormValues)
     .catch(error => {
+      dispatch(showNotification({
```

Comment:
I kindaaaaa wanna wait. Mocking out the forms is a little more involved and we already have tests for the notification actions, reducers and component so all we'd really be testing here is that it's called.